### PR TITLE
Add quality assurance tools

### PR DESCRIPTION
**Pull request by nvdbeek**
With this pull request it is possible to automatic testing builds

### CODE REVIEW

File: .github/workflows/qa.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.2
+          extensions: zip
```
Comment (dbrekelmans) [inline .github/workflows/qa.yml:17]: I don't think you need the `zip` extension in this repo.

**Review by dbrekelmans — APPROVED** [review]

**Comment by nusje2000** [comment]
I don't know why actions are not running, https://github.com/aeviiq/value-object/pull/8 seems to work. Maybe there is invalid syntax?

File: .github/workflows/qa.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+name: Quality Assurance
+
+on:
+  pull_request: null
+  push:
+    branches:
+      - master
```
Comment (nusje2000) [inline .github/workflows/qa.yml:7]:
```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
    branches: [ master ]
```

Maybe this is the reason why the builds are nog being executed?

File: phpstan.neon [hunk 3]
```diff
@@ -0,0 +1,10 @@
+parameters:
+	level: max
+	paths:
+		- src
+
+includes:
+    - vendor/phpstan/phpstan-strict-rules/rules.neon
+    - vendor/phpstan/phpstan-phpunit/extension.neon
+    - vendor/thecodingmachine/phpstan-safe-rule/phpstan-safe-rule.neon
+    - vendor/thecodingmachine/phpstan-strict-rules/phpstan-strict-rules.neon
```
Comment (nusje2000) [inline phpstan.neon:10]: There is a missing new line on the end of this file.
```suggestion
    - vendor/thecodingmachine/phpstan-strict-rules/phpstan-strict-rules.neon

```

File: phpunit.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        <testsuite name="default">
+            <directory>tests</directory>
+        </testsuite>
+    </testsuites>
+    <filter>
+        <whitelist>
+            <directory suffix=".php">./src</directory>
+        </whitelist>
+    </filter>
+</phpunit>
```
Comment (nusje2000) [inline phpunit.xml:21]: There is a missing new line on the end of this file.
```suggestion
</phpunit>

```

File: psalm.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+        xmlns="https://getpsalm.org/schema/config"
+        xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+>
+    <projectFiles>
+        <directory name="src"/>
+        <ignoreFiles>
+            <directory name="vendor"/>
+        </ignoreFiles>
+    </projectFiles>
+</psalm>
```
Comment (nusje2000) [inline psalm.xml:15]: There is a missing new line on the end of this file.
```suggestion
</psalm>

```

**Review by nusje2000 — CHANGES_REQUESTED** [review]
Everything looks okay, only some simple changes.

**Review by nusje2000 — APPROVED** [review]

**Pull request merged**